src/styles/blocks/main.scss

```diff
@@ -0,0 +1,40 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
Consider using relative units for the grid column width to improve responsiveness, especially for `grid-template-columns`.